cloudformation/cis-streams-and-vault.yml

```diff
@@ -0,0 +1,47 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Mozilla Change Integration Service Stack"
+Resources:
+  ### This is the stream of data flowing in from KPLs like Mozillians
+  CISInputStream:
+    Type: "AWS::Kinesis::Stream"
+    Properties:
+      ShardCount: 1
+      Tags:
+      - Key: Project
+        Value: ChangeIntegration
+      - Key: Type
+        Value: Input
+  ### DynamoDB Table of all users
+  IdentityVaultUsers:
+    Type: "AWS::DynamoDB::Table"
+    Properties:
+      AttributeDefinitions:
+      -
+        AttributeName: "UserId"
```

Comment:
I think that this one should be ``user_id`` after https://github.com/mozilla-iam/cis/pull/8/files#diff-16da26b86d8a5654e680f1d3757537f4R6